.github/workflows/release.yml

```diff
@@ -42,7 +42,6 @@ on:
         # Example provided for illustration, this value is derived by scripts/get-job-matrix.py build
         default: |
           {
-            "dotnet": "6.0.x",
```

Comment:
This should be retained, it's an example value and it's kept consistent across all of our workflows. If this is removed, someone might incorrectly believe the `dotnet` key isn't available here.